x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/reindex/use_reindex_state.tsx

```diff
@@ -27,20 +27,26 @@ export interface ReindexState {
   errorMessage: string | null;
   reindexWarnings?: ReindexWarning[];
   hasRequiredPrivileges?: boolean;
-}
-
-interface StatusResponse {
-  warnings?: ReindexWarning[];
-  reindexOp?: ReindexOperation;
-  hasRequiredPrivileges?: boolean;
+  meta: {
```

Comment:
nit: we could also pick this interface prop from `ReindexStatusResponse` to avoid duplication